components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -62,31 +64,59 @@ function StarcraftMatchSummary.MatchSummary(props)
 		:addClass('brkts-popup')
 		:addClass('brkts-popup-sc')
 		:addClass(match.opponentMode == 'uniform' and 'brkts-popup-sc-uniform-match' or 'brkts-popup-sc-team-match')
-		:node(StarcraftMatchSummary.Header({match = match}))
+		:node(StarcraftMatchSummary.Header({match = match, config = props.config}))
 		:node(StarcraftMatchSummary.Body({match = match}))
 		:node(StarcraftMatchSummary.Footer({match = match, showHeadToHead = match.headToHead}))
 end
 
 StarcraftMatchSummary.propTypes.Header = {
 	match = StarcraftMatchGroupUtil.types.Match,
+	config = 'table?'
 }
 
 function StarcraftMatchSummary.Header(props)
 	DisplayUtil.assertPropTypes(props, StarcraftMatchSummary.propTypes.Header)
 	local match = props.match
 
+	local showScore = (props.config or {}).showScore
+
 	local renderOpponent = function(opponentIx)
 		local opponent = match.opponents[opponentIx]
-		return StarcraftOpponentDisplay.BlockOpponent({
-			flip = opponentIx == 1,
+		local flip = opponentIx == 1
+		local opponentDisplay = {}
+		table.insert(opponentDisplay, StarcraftOpponentDisplay.BlockOpponent{
+			flip = flip,
 			opponent = opponent,
 			overflow = 'wrap',
 			showFlag = false,
-			showLink = opponent.type == 'team',
+			showLink = opponent.type == 'team' or showScore,
 		})
-			:addClass('brkts-popup-header-opponent')
-	end
+		local side = (flip and 'left' or 'right')
+
+		local display = html.create('div')
+
+		if showScore then
+			display:addClass('brkts-popup-header-opponent-' .. side)
+				:css('width', '50%')
+
+			local scoreDisplay = StarcraftOpponentDisplay.BlockScore{
+				isWinner = opponent.placement == 1 or opponent.advances,
+				scoreText = StarcraftOpponentDisplay.InlineScore(opponent),
+			}
+			scoreDisplay:addClass('brkts-popup-header-opponent-score-' .. side)
+			table.insert(opponentDisplay, scoreDisplay)
+		else
+			display:addClass('brkts-popup-header-opponent')
+		end
+		if not flip then
+			opponentDisplay = Array.reverse(opponentDisplay)
+		end
+		for _, item in ipairs(opponentDisplay) do
+			display:node(item)
+		end
```

Comment:
can also do `Array.extendWith(display.nodes, opponentDisplay)` if you're comfortable with manipulating fields directly on mw.html objects